comp/core/tagger/taggerimpl/collectors/workloadmeta_test.go

```diff
@@ -990,6 +990,42 @@ func TestHandleContainer(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "tags from environment with opentelemetry sdk",
```

Comment:
Should we also add a similar test in `TestHandleKubePod` ?

Reply:
Yes! Done 👍